packages/webapp/src/pages/_app.tsx

```diff
@@ -27,23 +27,18 @@ dayjs.extend(relativeTime.default);
 
 Modal.setAppElement("#__next");
 
-const WebApp = ({ Component, pageProps }: AppProps) => {
-    const queryClientRef = useRef<QueryClient>();
-    if (!queryClientRef.current) {
-        queryClientRef.current = new QueryClient();
-    }
-
-    return (
-        <QueryClientProvider client={queryClientRef.current}>
-            <Hydrate state={pageProps.dehydratedState}>
-                <EdenUALProvider>
-                    <Component {...pageProps} />
-                </EdenUALProvider>
-            </Hydrate>
-            <ReactQueryDevtools initialIsOpen={false} />
-            <Toaster />
-        </QueryClientProvider>
-    );
-};
+export const queryClient = new QueryClient();
```

Comment:
We can't do this. I think I sent you the reason @brandonfancher. From react query docs: 
> Create a new QueryClient instance inside of your app, and on an instance ref (or in React state). This ensures that data is not shared between different users and requests, while still only creating the QueryClient once per component lifecycle.

See more here: https://react-query.tanstack.com/guides/ssr#using-hydration

Reply:
You and I discussed it and wondered if the reason for making it a ref was for SSR...but we couldn't find that piece of documentation at the time, so we had concluded we would give this a try. (And I never circled back to run the CURL to see if SSR was working afterwards!)

I'll work with Mike this morning to adjust. I can think of two options:

1. In our named queries dependent on async functions using `queryClient`, do a `useQueryClient` and pass the instance into the async function.
2. Create an API instance within the app component in a ref and give it the `queryClient` instance. The async functions would be methods on that API instance and would therefore have access to the `queryClient`. (That's what I did in a previous production app.)

Option 1 is less intrusive and prob worth option for for now. 

Thoughts, @sparkplug0025?

Reply:
Ok, so out of our meeting this morning, I'm not making further changes here until we've researched the issue further.